Client/Frontend/Share/ShareExtensionHelper.swift

```diff
@@ -89,12 +89,8 @@ extension ShareExtensionHelper: UIActivityItemSource {
         return selectedURL
     }
 
-    // IMPORTANT: This method needs Swift compiler optimization DISABLED to prevent a nasty
-    // crash from happening in release builds. It seems as though the check for `nil` may
-    // get removed by the optimizer which leads to a crash when that happens.
-    @_semantics("optimize.sil.never") func activityViewController(_ activityViewController: UIActivityViewController, itemForActivityType activityType: UIActivityType) -> Any? {
-        // activityType actually is nil sometimes (in the simulator at least)
-        if activityType != nil && isPasswordManagerActivityType(activityType.rawValue) {
+    func activityViewController(_ activityViewController: UIActivityViewController, itemForActivityType activityType: UIActivityType) -> Any? {
```

Comment:
This method definition has changed (note the `?` for `UIActivityType`):

`func activityViewController(_ activityViewController: UIActivityViewController, itemForActivityType activityType: UIActivityType?) -> Any?`

Also note that we will still need to check if `activityType` is `nil`.

Reply:
That creates a warning that the signature doesn't match the SDK. I am going to rebase this on the 9.0 removal patch and see if the warning goes away.